specs/darwin/crash_logs.table

```diff
@@ -0,0 +1,27 @@
+table_name("crash_logs")
+description("Application Crash Logs")
+schema([
+    Column("process", TEXT, "Name of the process that crashed."),
+    Column("application_path", TEXT, "Path to the crashed process."),
+    Column("log_location", TEXT, "Location of log file."),
+    Column("identifier", TEXT, "Identifier of the crashed process."),
+    Column("version", TEXT, "Version info of the crashed process."),
+    Column("parent_proc", TEXT, "Parent of the crashed process."),
```

Comment:
I think that we usually just use `parent` for this in other tables. is this a pid?